packages/@uppy/react/src/useUppy.d.ts

```diff
@@ -1,5 +1,8 @@
 import type Uppy from '@uppy/core'
 
+/**
+ * @deprecated Initialize Uppy outside of the component or with `useState` and `useEffect` (see docs)
```

Comment:
```suggestion
 * @deprecated Initialize Uppy outside of the component (see docs)
```
Seems like we didn't find a way to make Uppy work properly, when initialised inside a component using useState/useEffect? (In a way that works with Strict Mode)